### ISSUE TITLE
.html is sort of fucked

### ISSUE THREAD

**Issue by filmaj** [xuijs/xui#15]
```
x$(document.body).html('bottom','<p id="this-is-a-test" style="font-size:12px;color:red;">hi</p>');
```

Results in:

```
<body>
    <p id="this-is-a-test style">hi</p>
</body>
```

Pretty sure it's something to do with wrap(). We should just rip out whatever jquery does and use that.

**Comment by ghost** [comment]
Hi 

I fixed this problem by changing the attrList regx in the wrap function 

attrList = result[2].split(/([A-Z]_\s_=\s_['|"][A-Z0-9:;#\s]_['|"])/i);

"[A-Z]_\s" to "[A-Z-]_\s"

**Comment by filmaj** [comment]
Thanks for the fix, committed and attributed you in 39afd48ffccba7eaa335c754d38fab8383353c55

Cheers!
